templates/how-it-works.html

```diff
@@ -4,59 +4,78 @@
 {% block page-title %} | How it works{% endblock %}
 
 {% block content %}
-<section class="row row-hero">
-    <div class="inner-wrapper equal-height">
-        <div class="seven-col">
-            <div class="align-vertically">
-                <h1>How it works</h1>
-                <p>MAAS has a tiered architecture with a central postgres database backing a &lsquo;Region Controller (regiond)&rsquo; that deals with operator requests. Distributed Rack Controllers (rackd) provide high-bandwidth services to multiple racks. The controller itself is stateless and horizontally scalable, presenting only a REST API.</p>
-                <p>Rack Controller (rackd) provides <abbr title="Dynamic Host Configuration Protocol">DHCP</abbr>, <abbr title="Intelligent Platform Management Interface">IPMI</abbr>, <abbr title="Preboot Execution Environment">PXE</abbr>, <abbr title="Trivial File Transfer Protocol">TFTP</abbr> and other local services. They cache large items like operating system install images at the rack level for performance but maintain no exclusive state other than credentials to talk to the controller.</p>
-            </div>
+<section class="p-strip--image is-dark is-deep" style="overflow: hidden; background-color: #333; background-image:url('https://assets.ubuntu.com/v1/6e4b2b20-MAAS-hero-background.png')">
```

Comment:
This is an edge case because the background image has transparency. Therefore we need to set this bespoke background colour.